Tabloid/client/src/components/CategoryList.js

```diff
@@ -25,8 +25,8 @@ const CategoryList = () => {
       <div className="container">
         <div className=" mt-4 row justify-content-center">
           {categories.map((category) => (
-            <ListGroupItem><Category category={category} key={category.id} />
-            <Link to={`/categories/edit/${category.id}`}>Edit</Link>{" "}
+            <ListGroupItem key={`editlink--${category.id}`}><Category category={category} key={category.id} />
```

Comment:
I believe this parent ListGroupItem component is the only one that requires a unique key. I could be wrong.. have you tested it with a key attribute only on this ListGroupItem component?